__tests__/app.test.js

```diff
@@ -43,8 +43,10 @@ describe("GET /api/reviews/:review_id", () => {
           category: expect.any(String),
           created_at: expect.any(String),
           votes: expect.any(Number),
+          comment_count: expect.any(String),
         };
         expect(reviewObject).toMatchObject(expectedReview);
+        expect(typeof +reviewObject.comment_count).toBe("number");
```

Comment:
as you're testing a known article, it's absolutely fine to expect a literal value here 👍